po/pl.po

```diff
@@ -858,31 +953,31 @@ msgstr ""
 "1. Przejdź do <b>Ustawień</b> → <b>Klawiatura</b> → <b>Wyświetl i dostosuj "
 "skróty</b> → <b>Własne skróty</b>.\n"
 "2. Kliknij przycisk <b>+</b>, aby utworzyć nowy skrót.\n"
-"3. Ustaw <b>Nazwę</b> na coś w rodzaju <i>Otwórz Zaawansowany zrzut ekranu</"
+"3. Ustaw <b>Nazwę</b> na coś w rodzaju <i>Otwórz Gradia</"
 "i>.\n"
 "4. Dla <b>Polecenia</b> skopiuj jedno z poniższych poleceń:"
 
-#: data/ui/preferences_window.blp:129
+#: data/ui/preferences_window.blp:173
 msgid "Interactive Screenshot"
 msgstr "Interaktywny zrzut ekranu"
 
-#: data/ui/preferences_window.blp:130
+#: data/ui/preferences_window.blp:174
 msgid "Allows you to select an area to screenshot"
 msgstr "Umożliwia wybranie obszaru do zrobienia zrzutu ekranu"
 
-#: data/ui/preferences_window.blp:148 data/ui/preferences_window.blp:177
+#: data/ui/preferences_window.blp:192 data/ui/preferences_window.blp:220
 msgid "Copy to clipboard"
 msgstr "Skopiuj do schowka"
 
-#: data/ui/preferences_window.blp:158
+#: data/ui/preferences_window.blp:201
 msgid "Full Screen Screenshot"
-msgstr "Zrzut ekranu całego okna"
+msgstr "Zrzut całego ekranu"
 
-#: data/ui/preferences_window.blp:159
+#: data/ui/preferences_window.blp:202
 msgid "Takes a screenshot of all screens instantly"
 msgstr "Wykonanie zrzut ekranu wszystkich ekranów natychmiast"
```

Comment:
I would inflect "Wykonanie" to "Wykonuje" and think about rewriting the latter section to remove the second "ekran", so that we don't repeat the same word in such a short sentence.